Makefile

```diff
@@ -34,9 +35,11 @@ setup:
 	docker-compose run specialist-publisher bundle exec rake db:seed
 	docker-compose run specialist-publisher bundle exec rake publishing_api:publish_finders
 	docker-compose run travel-advice-publisher bundle exec rake db:seed
+	docker-compose run manuals-publisher bundle exec rake db:seed
 	docker-compose run collections-publisher bundle exec rake db:setup
 	docker-compose run publisher bundle exec rake db:setup
 	docker-compose run frontend bundle exec rake publishing_api:publish_special_routes
+	docker-compose run whitehall bundle exec rake db:create db:purge db:setup
```

Comment:
If I remember correctly db:setup should do a drop and create so might not need create and purge steps

Reply:
Was running into this error when using just ` db:setup ` and there was already an existing database.

```
rake aborted!
ActiveRecord::StatementInvalid: Mysql2::Error: Cannot delete or update a parent row: a foreign key constraint fails: DROP TABLE `editions` CASCADE
```

Reply:
Ah I see, looking over the [rake file](https://github.com/rails/rails/blob/master/activerecord/lib/active_record/railties/databases.rake#L191-L192) it looks like I'm wrong on `db:setup` purpose it's `db:reset` that does drop first. Though I notice Whitehall [Jenkinsfile](https://github.com/alphagov/whitehall/blob/master/Jenkinsfile#L75) does steps indivudally but earliest commit doesn't indicate it was to get around a problem.